Utils/gitlab_triggers/trigger_content_internal_dist_nightly_build.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+
+_slack_channel="C04CHML16P8"
+
+while [[ "$#" -gt 0 ]]; do
+  case $1 in
+
+  -ct|--ci-token) _ci_token="$2"
+    shift
+    shift;;
+
+  -sdk|--sdk-ref) _sdk_ref="$2"
+    shift
+    shift;;
+
+  -ch|--slack-channel) _slack_channel="$2"
+    shift
+    shift;;
+
+  esac
+done
+
+if [ -z "$_ci_token" ]; then
+    echo "You must provide a ci token."
+    exit 1
+fi
+
+
+CI_SERVER_URL=${CI_SERVER_URL:-https://gitlab.xdr.pan.local} # disable-secrets-detection
+export BUILD_TRIGGER_URL="${CI_SERVER_URL}/api/v4/projects/1738/trigger/pipeline"
```

Comment:
```suggestion
export BUILD_TRIGGER_URL="${CI_SERVER_URL}/api/v4/projects/${GOLD_PROJECT_ID}/trigger/pipeline"
```

Reply:
maybe add line before:
GOLD_PROJECT_ID=${GOLD_PROJECT_ID:-1738} for manual triggering

Reply:
done